meminfo.h

```diff
@@ -5,35 +5,36 @@
 #define PATH_LEN 256
 
 #include <stdbool.h>
+#include <sys/types.h>
 
 typedef struct {
     // Values from /proc/meminfo, in KiB or converted to MiB.
-    long MemTotalKiB;
-    int MemTotalMiB;
-    int MemAvailableMiB; // -1 means no data available
-    int SwapTotalMiB;
-    long SwapTotalKiB;
-    int SwapFreeMiB;
+    size_t MemTotalKiB;
+    size_t MemTotalMiB;
+    ssize_t MemAvailableMiB; // -1 means no data available
+    size_t SwapTotalMiB;
+    size_t SwapTotalKiB;
+    size_t SwapFreeMiB;
     // Calculated percentages
-    int MemAvailablePercent; // percent of total memory that is available
-    int SwapFreePercent; // percent of total swap that is free
+    unsigned int MemAvailablePercent; // percent of total memory that is available
+    unsigned int SwapFreePercent; // percent of total swap that is free
 } meminfo_t;
 
 struct procinfo {
-    int pid;
-    int uid;
+    pid_t pid;
```

Comment:
Mismatch with the `printf`s in kill_largest_process

Reply:
Seems like there's no printf specifier specified for pid_t/uid_t.